.github/workflows/commit-check.yml

```diff
@@ -80,3 +89,25 @@ jobs:
         with:
           header: commit-message-lint-error
           delete: true
+
+      - name: "Mark as failed"
+        if: steps.check_commit_message.outputs.exit_code != 0
+        uses: actions/github-script@v7
+        with:
+          script: |
+            core.setFailed("Some commit messages are ill-formed")
+
+      - name: "Update breaking changes label"
+        if: always()
```

Comment:
lol

Reply:
Well this is actually needed!
Otherwise the job is skipped if anything previous failed 🤷